src/app/pages/details/common/get-this-title-files/amazon-associates-link.js

```diff
@@ -12,7 +12,8 @@ export default function useAmazonAssociatesLink(slug) {
                 if (urls.length > 0) {
                     setData(urls[0]);
                 }
-            });
+            })
+            .catch((err) => console.error(`Fetching buyprint: ${err}`));
```

Comment:
do you have sentry setup? if so should this log to it?

Reply:
We do have Sentry set up. Since this catches the error, it won't log to Sentry. I don't think I want it to, since it's not something I can fix if Buyprint is down.